pymtl3/passes/sverilog/import_/test/ImportedObject_test.py

```diff
@@ -76,3 +82,47 @@ def construct( s ):
   a._tv_in = tv_in
   a._tv_out = tv_out
   do_test( a )
+
+def test_normal_queue( do_test ):
+  def tv_in( m, tv ):
+    m.enq_en = Bits1( tv[0] )
+    m.enq_msg = Bits32( tv[1] )
+    m.deq_en = Bits1( tv[3] )
+  def tv_out( m, tv ):
+    if tv[2] != '*':
+      assert m.enq_rdy == Bits1( tv[2] )
+    if tv[4] != '*':
+      assert m.deq_rdy == Bits1( tv[5] )
+    if tv[5] != '*':
+      assert m.deq_msg == Bits32( tv[4] )
+  class VQueue( Placeholder, Component ):
+    def construct( s, data_width, num_entries, count_width ):
+      s.count   =  OutPort( mk_bits( count_width )  )
+      s.deq_en  =  InPort( Bits1  )
+      s.deq_rdy = OutPort( Bits1  )
+      s.deq_msg = OutPort( mk_bits( data_width ) )
+      s.enq_en  =  InPort( Bits1  )
+      s.enq_rdy = OutPort( Bits1  )
+      s.enq_msg =  InPort( mk_bits( data_width ) )
+      s.sverilog_import = ImportConfigs(vl_src = get_dir(__file__)+'VQueue.sv')
```

Comment:
if it's the same dir, can the user uses 'VQueue.sv' by default?

Reply:
Most of the cases we would like to run this in a build directory and we seldom put source files under a build directory. But if somebody does this, they can use the filename alone.

Reply:
good point. Running this in build directory is very likely.